### PR TITLE
Adds a flag to control persistance in database

### PR DESCRIPTION
**Pull request by Sornii**
The ability to control if I wan't to persist the log in database.

### CODE REVIEW

**Comment by Sornii** [comment]
It's needed to add in documentation the option if accepted.

**Comment by Sornii** [comment]
@zol @TheGame2500 could any of you approve this?

**Comment by TheGame2500** [comment]
@Sornii done, good job. @zol could you please update the package on atmosphere.

File: synced-cron-tests.js [hunk 1]
```diff
@@ -227,3 +227,16 @@ Tinytest.add('Single time schedules don\'t break', function(test) {
...
+  SyncedCron._reset();
+
+  var testEntryNoPersist = _.extend({}, TestEntry, {persist: false});
+
+  SyncedCron.add(testEntryNoPersist);
+
+  const now = new Date();
+  SyncedCron._entryWrapper(testEntryNoPersist)(now);
+  test.equal(SyncedCron._collection.find().count(), 0);
+});
```
Comment (Sornii) [inline synced-cron-tests.js]: My mistake here, it should have a empty newline, I'll fix it and add this flag to the docs.

**Comment by rclai** [comment]
Guys, this feature hasn't been published to Atmosphere.

**Comment by lmachens** [comment]
Good PR!
But still not published on Atmosphere. @zol?

**Pull request merged**